### ISSUE TITLE
Certificate should include the hostname

### ISSUE THREAD

**Issue by OzTK** [wking-io/elm-serve#9]
Passing the `--hostname` and requesting `--ssl` results in a certificate not matching the hostname (it's a blocker to access the server on Chrome).
I would suggest making the hostname the CommonName of the cert, but I'm not an expert and maybe it should be part of the `altnames`?

Happy to open a PR once this is clarified 😃

**Comment by wking-io** [comment]
Yes I noticed that recently when working with ssl. I was not familiar with how it worked and someone else actually implemented it. I will try and figure it out as well and if you figure it out first I would love a PR!

**Comment by wking-io** [comment]
It will actually get bundled over into the new update of elm-live. I am not going to remove elm-serve, but the server was causing too many issues being separate from the elm-live repo

**Comment by OzTK** [comment]
Understood. Does this mean this issue will be fixed in that elm-live update? If not, I could open a PR on elm-live when the server is migrated in

**Comment by wking-io** [comment]
If you want to make it here that would be perfect and then we can work it into the release and both versions will be fixed :)
